odoo/addons/base/models/res_partner.py

```diff
@@ -340,6 +344,11 @@ def _compute_commercial_company_name(self):
             p = partner.commercial_partner_id
             partner.commercial_company_name = p.is_company and p.name or partner.company_name
 
+    def _compute_company_registry(self):
+        # exists to allow overrides
+        for company in self:
+            company.company_registry = company.company_registry
```

Comment:
This is a `res.partner`
Also, a `pass` statement would maybe remover the additional queries in the test?
You don't need to assign anything for stored fields

Reply:
Kept it as-is to avoid too much diff.
Replaced it with pass, but it doesn't change anything regarding the queries :/